src/LTO/Accounts/AccountFactoryECDSA.py

```diff
@@ -11,6 +11,7 @@ class AccountECDSA(AccountFactory):
     def __init__(self, chainId, curve='secp256k1'):
         super().__init__(chainId)
 
+        self.keyType = curve
         if curve == 'secp256k1':
```

Comment:
Should throw an error if something else than `secp256k1` or `secp256r1` is given as curve.